zio-http/src/test/scala/zhttp/http/EncodeClientRequestSpec.scala

```diff
@@ -38,13 +38,13 @@ object EncodeClientRequestSpec extends DefaultRunnableSpec with EncodeClientPara
       testM("uri") {
         check(anyClientParam) { params =>
           val req = encodeClientParams(HttpVersion.HTTP_1_1, params)
-          assert(req.uri())(equalTo(params.url.asString))
+          assert(req.uri())(equalTo(params.url.relative.asString))
         }
       } +
       testM("uri on HttpData.File") {
         check(HttpGen.clientParamsForFileHttpData()) { params =>
           val req = encodeClientParams(HttpVersion.HTTP_1_1, params)
-          assert(req.uri())(equalTo(params.url.asString))
+          assert(req.uri())(equalTo(params.url.relative.asString))
```

Comment:
Let's tests with actual and expected urls. Here we are testing implementation details.